app/components/LargeHeader.js

```diff
@@ -1,8 +1,11 @@
 import React from 'react';
 
+let bgImage = require('../assets/images/bg.png');
```

Comment:
Using `require` tells webpack that the image is a dependency, so it will a) copy it to the output directory for us and b) return the public path to the image, so we don't have to hard-code it as we used to.